app/routes/pages.js

```diff
@@ -10,10 +10,16 @@ module.exports = function() {
   router.get('/settings.json', settings.main);
 
   router.get('/', pages.main);
-  router.get(/embed\/(.*)/, function(req, res, next) {
+
+  router.get(/embed\/(treemap|piechart|barchart|linechart|bubbletree|table|map)\/(.*)/, function(req, res, next) {
```

Comment:
@pwalsh every views from babbage.ui require different initial parameters. We have hardcode for charts and tabs even in babbage.ui.